mobile/native/src/trade/order/handler.rs

```diff
@@ -17,7 +18,7 @@ pub async fn submit_order(order: Order) -> Result<()> {
     let url = format!("http://{}", config::get_http_endpoint());
     let orderbook_client = OrderbookClient::new(Url::parse(&url)?);
 
-    if let Err(e) = position::handler::update_position_after_order_submitted(order) {
+    if let Err(e) = position::handler::get_position_matching_order(&order) {
```

Comment:
```suggestion
    if let Err(e) = position::handler::does_order_close_position(&order) {
```

I would love to have a name that includes the purpose of this call - "check if there is a position and the order is closing it". If you don't think that helps feel free to ignore this comment.

Reply:
IMHO the problem was there beforehand, and I just uncovered it.